network-api/networkapi/templates/fragments/buyersguide/pni_mobile_search.html

```diff
@@ -7,11 +7,13 @@
                 <img
                     class="tw-w-8 tw-h-8 tw-mr-4"
                     src="{% static "_images/buyers-guide/pni-search.svg" %}"
+                    alt=""
                 >
                 <input
                     id="pni-mobile-bar"
                     type="text"
                     class="tw-w-full tw-bg-blue-05 tw-outline-none tw-py-7 tw-text-base tw-text-blue-60 placeholder:tw-text-blue-60"
+                    name="search"
```

Comment:
For better announcement when the placeholder is overridden / a value is entered.